san/graphql.py

```diff
@@ -38,6 +38,14 @@ def __handle_success_response__(response, gql_query_str):
                 gql_query_str))
 
 
+def get_all_available_metrics():
```

Comment:
Can you reuse `execute_gql` here ? Also I think this function does not belong to this file but to file with other queries.